article/test/ArticleAmpValidityTest.scala

```diff
@@ -4,6 +4,11 @@ import org.scalatest.DoNotDiscover
 
 
 @DoNotDiscover class ArticleAmpValidityTest extends AmpValidityTest {
+
+  /*
+      NB: these may be duplicated in tools/amp-validation - if you add or remove
+      a URL from here, think about whether it should be removed there too.
+   */
```

Comment:
As discussed, we're expecting the 2 sets of test endpoints to diverge over time.  Have updated comments to match (even if no one reads them 😁 )